about.html

```diff
@@ -12,62 +12,82 @@
 </head>
 
 <body>
+  <header>
+    <div class='container'>
+    <div class='top'>
+        <div class='logo'>
+            <img class='logo1' src='img/lambda-black.png' alt="">
```

Comment:
Always fill out the alt="", in the event an image can't load the user can still see what the image was supposed to be.